docs/3.0.0-beta.x/plugin-development/frontend-development.md

```diff
@@ -71,10 +71,10 @@ Here are its properties:
 | isReady                   | boolean | The app will load until this proprety is true                                                           |
 | leftMenuLinks             | array   | `[]`                                                                                                    |
```

Comment:
If the third column is a description, you will need to update the `leftMenuLinks`, `blockerComponent`, and `blockerComponentProps` description.